Modules/Devices/test/Devices.API.Tests.Integration/StepDefinitions/GetIdentityListStepDefinitions.cs

```diff
@@ -0,0 +1,45 @@
+using Challenges.API.Tests.Integration.Models;
+using Devices.API.Tests.Integration.API;
+using Devices.API.Tests.Integration.Models;
+
+namespace Devices.API.Tests.Integration.StepDefinitions
+{
+    [Binding]
+    public class GetIdentityListStepDefinitions
+    {
+        private readonly RequestConfiguration _requestConfiguration;
+        private readonly IdentitiesApi _identitiesApi;
+        private HttpResponse<IdentityResponse>? _identitiesResponse;
+        private List<IdentitySummaryDTO>? _identitiesList;
+
+        public GetIdentityListStepDefinitions(IdentitiesApi identitiesApi)
+        {
+            _identitiesApi = identitiesApi;
+            _requestConfiguration = new RequestConfiguration();
+        }
+
+   
+        [When(@"a GET request is sent to the Identities/ endpoint")]
+        public async Task WhenAGETRequestIsSentToTheIdentitiesEndpointAsync()
+        {
+            _identitiesResponse = await _identitiesApi.GetIdentitiesList(_requestConfiguration);
+            _identitiesResponse.Should().NotBeNull();
+            _identitiesResponse!.Data.Should().NotBeNull();
+            _identitiesList = _identitiesResponse!.Data!.Result;
+        }
+
+        [Then(@"the response status code is (\d*) \((?:[a-z]|[A-Z]|\s)+\)")]
```

Comment:
Could you change the regex as follows (for the other integration tests as well please)
```suggestion
        [Then(@"the response status code is (\d\d\d) \((?:[a-z]|[A-Z]|\s)+\)")]
```